### PR TITLE
update code of conduct to include weapons policy

### PR DESCRIPTION
**Pull request by VeldaKiara**
This is a revised and updated version of the code of conduct.

The format of some of the titles has been changed

A new addition of the weapons policy as agreed has been included.

### CODE REVIEW

File: src/about/conduct.html [hunk 1]
```diff
@@ -106,7 +106,32 @@ <h2>Local authorities</h2>
...
+-   Pattern of inappropriate social contact, such as requesting/assuming inappropriate levels of intimacy with others
+-   Sustained disruption of online community discussions, in-person presentations, or other in-person events
+-   Continued one-on-one communication after requests to cease
+-   Other conduct that is inappropriate for a professional audience including people of many different backgrounds
+
+This list is not an exhaustive list of all inappropriate behavior that is unacceptable at DjangoCon US.
+
+If you are unsure whether the incident is a violation, or whether the space where it happened is covered by the Code of Conduct, we encourage you to still report it. 
+
+We are fine with receiving reports where we decide to take no action for the sake of creating a safer space.
```
Comment (tim-schilling) [inline src/about/conduct.html]: I'd prefer to see this above the list of examples so the person knows that before they try finding a relatable scenario.

**Review by tim-schilling — APPROVED** [review]
Generally in favor of the changes, thank you! I'm unsure if we should be committing the changes to settings.json and package-lock.json as a part of this

File: src/about/conduct.html (hunk 1 above)
Comment (VeldaKiara) [reply src/about/conduct.html]: checking on that in a minute

**Review by sodevious — CHANGES_REQUESTED** [review]
can we please restore `package_lock.json`? the `.gitignore` changes should also be reverted- we need these files tracked. i know `package_lock.json` can change on your filesystem if you `npm install`, which is fine/normal but instead could the file not be added to the commit/PR?

**Pull request merged**